packages/shared/src/graphql/feed.ts

```diff
@@ -183,8 +183,13 @@ export const BOOKMARKS_FEED_QUERY = gql`
     $loggedIn: Boolean! = false
     $first: Int
     $after: String
+    $supportedTypes: [String!] = ["article", "share"]
```

Comment:
@el It's actually better to fix it on the backend side I think.
Not sure @idoshamun agrees.

Reply:
We can revert the change once the backend is updated in that case